#diffx: encoding=utf-8, version=1.0
#.change:
#..file:
#...meta: length=138
path: "/src/message.py"
revision:
    old: "9f6a412b3aee0a55808928b43f848202b4ee0f8d"
    new: "f814cf74766ba3e6d175254996072233ca18a690"
#...diff: length=631
--- a/src/message.py
+++ b/src/message.py
@@ -164,10 +164,10 @@
             not isinstance(headers, MultiValueDict)):
             # Instantiating a MultiValueDict from a dict does not ensure that
             # values are lists, so we have to ensure that ourselves.
-            headers = MultiValueDict(dict(
-                (key, [value])
-                for key, value in six.iteritems(headers)
-            ))
+            headers = MultiValueDict({
+                key: [value]
+                for key, value in headers.items()
+            })

         if in_reply_to:
             headers['In-Reply-To'] = in_reply_to
